koku/masu/util/aws/common.py

```diff
@@ -518,11 +567,27 @@ def scrub_resource_col_name(res_col_name):
     return column_dict.apply(json.dumps), unique_keys
 
 
+def check_aws_custom_columns(col_names):
+    REQUIRED_COLS = set(utils.CSV_REQUIRED_COLUMNS["AWS-custom"])
+    missing_cols = False
+    if not set(col_names).issuperset(REQUIRED_COLS):
+        missing_cols = True
+    return missing_cols, REQUIRED_COLS
+
+
 def aws_post_processor(data_frame):
     """
     Consume the AWS data and add a column creating a dictionary for the aws tags
     """
+    org_columns = set(list(data_frame))
+    columns = []
+    for col in org_columns:
+        if "/" not in col:
+            if COL_TRANSLATION.get(col):
+                data_frame = data_frame.rename(columns={col: COL_TRANSLATION[col]})
+                columns.append(COL_TRANSLATION[col])
     columns = set(list(data_frame))
+    columns = set(columns)
     columns = set(TRINO_REQUIRED_COLUMNS).union(columns)
```

Comment:
Set methods accept an iterable, so the previous two lines could be removed in favor of 
```suggestion
    columns = set(TRINO_REQUIRED_COLUMNS).union(data_frame)
```